rb_ws/src/buggy/buggy/simulator/engine.py

```diff
@@ -51,43 +44,43 @@ def __init__(self):
         self.steering_subscriber = self.create_subscription(
```

Comment:
Issue with self.pose_publisher, that we didn't discuss yet. The INS output now is in ECEF, so either we simulate that or provide clean_states, which are specified as part of the wiki page

Reply:
published as clean_slate